redwood-protocol-guest/src/commonMain/kotlin/app/cash/redwood/protocol/guest/ProtocolRedwoodComposition.kt

```diff
@@ -30,6 +31,7 @@ import kotlinx.coroutines.flow.StateFlow
  * @param scope A [CoroutineScope] whose [coroutineContext][kotlin.coroutines.CoroutineContext]
  * must have a [MonotonicFrameClock] key which is being ticked.
  */
+@OptIn(RedwoodCodegenApi::class)
 @Suppress("FunctionName")
 public fun ProtocolRedwoodComposition(
```

Comment:
This function now has a type in its public API that is reserved for codegen use only. Ideally you should be able to setup a protocol-based system relying on public API only. Maybe something fixed in future change.

Reply:
I think I can do that. ‘Set up a protocol-based system with the public API only’ is a nice standard to reach for.